form.schema.json

```
@@ -0,0 +1,216 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "http://json.openmrs.org/form.schema.json",
+  "$comment": "This schema is intended to guide authors of forms built for use in O3",
+  "title": "O3 Form schema",
+  "description": "This schema defines the expected values of a form schema built using the O3 Form Builder. These schemas are used to build forms for use in O3. The schema is based on the JSON Schema Draft 7 specification. For more information, see https://json-schema.org/specification.html",
+  "type": "object",
+  "properties": {
+    "$schema": {
+      "type": "string",
+      "description": "The schema for this JSON document. Should point to the canonical URL for this document.",
+      "default": "http://json.openmrs.org/form.schema.json"
+    },
+    "name": {
+      "type": "string",
+      "description": "Name of the form"
+    },
+    "uuid": {
+      "type": "string",
+      "description": "Unique identifier of the form. Autogenerated by the system"
```

This is actually somewhat significant on the Iniz side; we assume anything where this is "XXXX" should have an autogenerated UUID, but that otherwise the value is the expected UUID.

```suggestion
      "description": "Unique identifier of the form. Autogenerated by the system"
      "default": "XXXX"
```